packages/shared/components/MenuSshLogin/MenuSshLogin.tsx

```diff
@@ -120,6 +120,7 @@ export const LoginItemList = ({ logins, onClick, onKeyPress }) => {
         type="text"
         autoFocus
         placeholder="Enter login name..."
+        autocomplete="off"
```

Comment:
i was able to reproduce this (strangely only pop ups for me in console nodes list, but the change still applies b/c it uses same component)
```suggestion
        autoComplete="off"
```
with react, attributes need to be camel cased for it to work: https://reactjs.org/docs/dom-elements.html